bundles/org.openhab.binding.boschindego/src/main/java/org/openhab/binding/boschindego/internal/IndegoController.java

```diff
@@ -12,19 +12,19 @@
  */
 package org.openhab.binding.boschindego.internal;
 
+import java.net.http.HttpClient;
 import java.time.Instant;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
-import org.eclipse.jetty.client.HttpClient;
```

Comment:
I thought the use of the common Jetty HttpClient was required?

Reply:
Yeah needed some time to fix all the imports and grab a coffie. Fixed now.